index.js

```diff
@@ -28,13 +29,91 @@ module.exports = function (chai, utils) {
   Assertion.overwriteMethod('eql', assertEql);
   Assertion.overwriteMethod('eqls', assertEql);
 
-  function explicitRoughly(tolerance) {
-    utils.flag(this, 'tolerance', tolerance);
+  function checkComparator(comparatorOrTolerance) {
+    if ('number' === type(comparatorOrTolerance)) {
+      return createRelativeComparator(comparatorOrTolerance);
+    } else if ('function' === type(comparatorOrTolerance)) {
+      return comparatorOrTolerance;
+    } else {
+      throw new Error('Roughly must be given a relative tolerance number or comparator function');
+    }
+  }
+
+  function explicitRoughly(comparatorOrTolerance) {
+    utils.flag(this, 'comparator', checkComparator(comparatorOrTolerance));
   }
 
   function defaultRoughly() {
-    utils.flag(this, 'tolerance', DEFAULT_TOLERANCE);
+    utils.flag(this, 'comparator', createRelativeComparator(DEFAULT_TOLERANCE));
   }
 
   Assertion.addChainableMethod('roughly', explicitRoughly, defaultRoughly);
 };
+
+// copied from deep-eql
+function simpleEqual(a, b) {
+  // Equal references (except for Numbers) can be returned early
+  if (a === b) {
+    // Handle +-0 cases
+    return a !== 0 || 1 / a === 1 / b;
+  }
+
+  // handle NaN cases
+  return (
+    a !== a && // eslint-disable-line no-self-compare
+    b !== b // eslint-disable-line no-self-compare
+  );
+}
```

Comment:
Hi, the default comparator is here:
https://github.com/Turbo87/chai-roughly/pull/7/files#diff-168726dbe96b3ce427e7fedce31bb0bcR47

That is, created by `createRelativeComparator`, which uses `relativeTest`.  

This is basically the same as before.  BUT, sorry, I forgot to mention, I propose a slight change:  it is now symmetric, so that comparing (a, b) will give same result as (b, a), which I believe wasn't the case before.  (Previously the test was done relative to b, now it is done relative to greater of absolute value of b, and absolute value of a).  In most cases that will be immaterial difference.